ts/features/itwallet/presentation/remote/machine/actions.ts

```diff
@@ -32,6 +32,15 @@ export const createRemoteActionsImplementation = (
     });
   },
 
+  navigateToIdentificationModeScreen: () => {
+    navigation.navigate(ITW_ROUTES.MAIN, {
+      screen: ITW_ROUTES.IDENTIFICATION.MODE_SELECTION,
+      params: {
+        eidReissuing: true
+      }
+    });
+  },
+
   closeIssuance: () => {
```

Comment:
I think it's a typo that we forgot to fix :) Please can do it?
```suggestion
  close: () => {
```